Lib/argparse.py

```diff
@@ -1540,6 +1540,19 @@ def _get_positional_kwargs(self, dest, **kwargs):
             msg = _("'required' is an invalid argument for positionals")
             raise TypeError(msg)
 
+        # make sure 'store_true', 'store_false', or 'store_const' action
+        # is not specified
+        if 'action' in kwargs:
+            if kwargs.get('action') == 'store_true':
```

Comment:
All the code above can be simplified to:

```python
action = kwargs.get('action')
if action in ('store_true', 'store_false', 'store_const'):
    msg = _("{action} is an invalid action for positionals", {'action': action})
    raise TypeError(msg)
```

Reply:
Thanks! I knew I was still away from a more pythonic version.